### ISSUE TITLE
Match up window buttons

### ISSUE THREAD

**Issue by thany** [adobe/brackets#12320]
It makes sense to match up the window buttons with other applications. This makes it easier to minimize a couple of windows one after another, Brackets being one of them.

As you can see:
![buttons](https://cloud.githubusercontent.com/assets/152227/13960981/d31edeca-f05a-11e5-8bba-93e8b8910630.png)
That's not currently the case.

That's Windows 10 btw. The buttons probably line up differently on different OS'es, but it's important for usability. The other option is: why theme the windows chrome in the first place? The user has set a certain theme (and colour) that is not respected.

**Comment by danvim** [comment]
@thany I personally don't see that as a problem. In fact, the smaller menu bar gives more space to code with. :confused:

**Comment by thany** [comment]
It's not the smaller title bar, is the horizontal position of the window buttons.

So, if multiple windows are maximized & overlapping, we cannot simply click a bunch of times on the minimize buttons to minimize them all, because the one in Brackets is misaligned.

**Comment by thany** [comment]
Question: why was this issue closed? The problem is not solved...

**Comment by danvim** [comment]
@thany Perhaps the issue already exist in the one in the reference?

**Comment by thany** [comment]
Any idea who closed this issue, and why??

**Comment by ficristo** [comment]
AFAICT this issue is still open.
The linked one is closed.

**Comment by thany** [comment]
Really? Oh well, sorry then. I looked that way to me.

**Comment by thany** [comment]
Also, since 1.10 the window buttons look even more off. Especially when Brackets isn't maximised, they look completely misplaced.

**Comment by ficristo** [comment]
Yeah, release 1.10 was built with a newer compiler which caused https://github.com/adobe/brackets/issues/12430
